### DISCUSSION TITLE
CSS | HTML - modifiche non visibili

### DISCUSSION THREAD

**Discussion by LorenzoPetraroli** [globaleaks/globaleaks-whistleblowing-software#3570] [General]
Buongiorno, 
dopo aver completato la configurazione, 
per testare ho provato a modifica nel css dell header, ma le modifiche non sono visibili e resta il codice di default

Dopo aver fatto le modifiche sul file ho seguito questi passaggi:
1) grunt build" nella directory client  
2) la procedura di avvio del server

le modifiche sdono presenti nel file "styles.css" situato in Globaleaks/client/build/css
Vorrei sapere se i file sono modificabili direttamente dal codice o esclusivamente dall'applicazione come admin user.

nella console del browser ho questo messaggio: Error while fetching an original source: request failed with status 404
Source URL: https://127.0.0.1:8443/css/build/css/styles.css

Lo stesso discorso vale per html.

Grazie in anticipo 
Lorenzo

**Comment by evilaliv3** [comment]
Grazie della domanda.

riuscite a riprodurre il problema su try.globaleaks.org?  potete condividere il css che state provando ad applicare?

In realtà non è previsto che globaleaks venga modificato da sorgenti (per ragioni di sicurezza) ma se volete indicarci come procedere possiamo supportarvi ad ottenere il risultato che volete.

**Reply by LorenzoPetraroli** [reply]
Grazie mille per la risposta velocissima,
ho appena iniziato a lavorarci su questa piattaforma quindi perdona la mia poca esperienza.

Quindi in sostanza io non posso modificare i file che ho nella directory e fare la buil, ma posso solo agire dalle impostazioni una volta effettuato il login da admin?

Ho controllato su try.globaleaks.org e l'errore in console è solo presente su mozzilla, mentre su chrome no

**Comment by evilaliv3** [comment]
Corretto. Potere puoi certamente ma non è raccomandabile.

L'applicazione offre le sue garanzie quando usata nella sua versione originale che viene verificata in un processo di peer review continuo.

**Reply by LorenzoPetraroli** [reply]
Tutto chiaro, allora rimane solo il problema in origine:

Ho inserito questa modifica nel file main.css
`#HeaderBox {
  color: #FFF;
  background: #428bca;
  border-bottom: 1px solid #357ebd;
}`

Avresti gentilmente delle indicazioni o qualche procedimento per provare a visualizzare le modifiche?
Grazie ancora per l'aiuto prezioso

**Comment by evilaliv3** [comment]
La configurazione è corretta.

La tua è una configurazione single tenant o multi tenant?

**Reply by LorenzoPetraroli** [reply]
Single tenant, ho seguito puramente la procedura indicata nella sezione "Environment"

**Comment by LorenzoPetraroli** [comment]
Il problema è stato risolto, sulla macchina era stato installato anche la verrsione con wget, quindi andava in conflitto, e il css veniva preso dalla cartella dell'installazione

grazie mille per il supporto
